docs/HowTo/Use-Privacy/Run-Tessera-With-Besu.md

```diff
@@ -24,6 +24,8 @@ and [run in a separate instance](#separate-instances) to Hyperledger Besu.
 ## High availability
 
 Privacy requires you to [configure Tessera for high availability].
+Tessera also requires [`orion` mode](https://docs.tessera.consensys.net/en/stable/HowTo/Configure/Orion-Mode/)
+to be enabled if **not** running Besu in [GoQuorum compatible privacy mode](../../HowTo/Use-Privacy/Use-GoQuorum-compatible-privacy.md).
```

Comment:
this one I have to check. I think it is more correct to say 
You need to run Tessera in orion mode only if you are using Besu's flexible privacy groups.